contentScript.js

```diff
@@ -60,6 +38,32 @@ function appendAccessibilityInfo() {
   });
 }
 
+function validateImage(image) {
+  const altText = image.getAttribute('alt') ? image.getAttribute('alt').trim() : "";
+  const parent = image.closest('a') || image.closest('button');
+  if (!parent) return
+  const parentAriaLabel = parent.getAttribute('aria-label') && parent.getAttribute('aria-label').trim()
```

Comment:
currently GitHub doesn't support `aria-label` at all within markdown but the GIF player we render does have an `aria-label` so this accounts for that... as well as future changes, potentially

Reply:
nice!